html-forms/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="#" method="post">
+      <div>
+        <p>
+          <label for="email">Email</label>
+        </p>
+        <input type="email" name="email" id="email" />
+      </div>
+
+      <div>
+        <p>
+          <label for="dropdown">Why is coding cool?</label>
+        </p>
+        <select name="dropdown" id="dropdown">
+          <option selected hidden>Select an option</option>
+          <option value="option-one">Because it's interesting</option>
+          <option value="option-two">Because it's fun</option>
+          <option value="option-three">Because it's powerful</option>
```

Comment:
FYI, the string you insert in the `value` attribute is what will get sent to the server on form submission - it may make more sense to say something like `value="interesting"` or `value="fun"`